lib/rspecq/reporter.rb

```diff
@@ -131,15 +133,18 @@ def flaky_jobs_to_sentry(jobs, build_duration)
 
       jobs.each do |job|
         filename = job.sub(/\[.+\]/, "")[%r{spec/.+}].split(":")[0]
+        seed = @queue.job_metadata(job, "seed")
+        failure_reason = @queue.job_metadata(job, "failure_reason")
+        backtrace = @queue.job_metadata(job, "backtrace")
+        worker_id = @queue.job_metadata(job, "worker_id")
 
         extra = {
           build: @build_id,
           build_timeout: @timeout,
-          queue: @queue.inspect,
-          object: inspect,
-          pid: Process.pid,
```

Comment:
Given we're in the context of a monitoring tool, these things are good to have around since they might prove useful for debugging (that's why they were added in the first place). And there's effectively zero cost in submitting them.